mobile/src/main/java/org/openhab/habdroid/model/Item.kt

```diff
@@ -175,6 +169,15 @@ fun JSONObject.toItem(): Item {
         }
     }
 
+    fun JSONObject.getDoubleOrNull(name: String) = if (has(name)) {
```

Comment:
This function should be placed next to `JSONObject.optStringOrNull()` and be renamend to `JSONObject.optDoubleOrNull`.